frontend-runitup/src/Components/Authentication/LoginPage.jsx

```diff
@@ -0,0 +1,99 @@
+import "../../styles/LoginPage.css";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+function LoginPage({ onLogin }) {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const navigate = useNavigate();
+
+  const handleChangeUser = (e) => {
+    setUsername(e.target.value);
+  };
+
+  const handleChangePassword = (e) => {
+    setPassword(e.target.value);
+  };
+
+  const handleLogin = (e) => {
+    e.preventDefault();
+
+    fetch(`${import.meta.env.VITE_BACKEND_ADDRESS}/login`, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        username,
+        password,
+      }),
+    })
+      .then((response) => {
+        if (response.ok) {
+          return response.json();
+        } else {
+          throw new Error("Failed to login");
+        }
+      })
+      .then((data) => {
+        onLogin({ name: username }, data.accessToken, data.refreshToken);
+        navigate("/feed");
+      })
+      .catch((error) => {
+        console.error("Error:", error);
+      });
```

Comment:
Ok for development, but make sure this doesn't go into the prod. Logging error into console might reveal private info about the user or website

Reply:
Thank you for pointing that out. I will go and fix that and make sure to remove those console messages when going into prod!